articles/azure/azs-how-create-availability-set.md

```diff
@@ -0,0 +1,166 @@
+
+Let’s consider a typical VM-based solution where you might have four front-end web servers and 2 back-end VMs hosting databases. With Azure Stack, you’d want to define two availability sets before you deploy your VMs: one availability set for the web tier and one availability set for the back tier. When you create a new VM you can then specify the availability set as a parameter to the `az vm` create command, and Azure Stack automatically ensures that the VMs you create within the available set are isolated across multiple physical hardware resources. If the physical hardware that one of your Web Server or back-end VMs is running on has a problem, you know that the other instances of your Web Server and back-end VMs remain running because they are on different hardware.
+
+## Creating an availability set via the portal
+
+1. The first step when initially creating an availability set is to create the resource group for the availability set.
+
+    ![Microsoft Azure availability set step 1](images/azs-portal-create-availability-set1.png)
+
+2. Name the resource group.
+
+    ![Microsoft Azure availability set step 2](images/azs-portal-create-availability-set2.png)
+
+3. Once the resource group has been created if you navigate into it and select **Overview** then you will be able to select **Create resources**.
+
+    ![Microsoft Azure availability set step 3](images/azs-portal-create-availability-set3.png)
+
+4. This will load the marketplace, if you search for **Availability Set**, you will be given the option to create one.
+
+    ![Microsoft Azure availability set step 4](images/azs-portal-create-availability-set4.png)
+
+5. This will bring up the **Create availability** set blade. Here you can set the name of the set and how many fault and update domains you want it to have. The resource group should be the same as the one you created earlier. As for the option for managed disks, select the option for which matches the VMs disks you will be adding to the set. Once done select **Create** and the availability set will be deployed.
+
+    ![Microsoft Azure availability set step 5](images/azs-portal-create-availability-set5.png)
+
+6. Now when you are creating the VM's you want to be a part of the availability set, when you get to step 3, under **High Availability** select the availability set you want to use. Make sure to set the disk type of the VM first.
```

Comment:
Remove Now from the beginning of the sentence. Change you are to you're. Remove the apostrophe from VMs.

Reply:
done